src/styles/ui/CardButton.scss

```diff
@@ -36,19 +53,16 @@
     }
   }
 
-
   box-shadow: -2px 6px 10px 0px rgba(0,0,0,0.49);
-  // text-align: center;
   align-items: flex-start;
   display: inline-flex;
   font-weight: 700;
   text-transform: uppercase;
-  // font-size: 13px;  // original
-  font-size: 20px;
+  font-size: 16px;
   text-align: left;
   padding: 0.7em;
-  height: 225px;
-  width: 160px;
+  height: 230px; //225px;
+  width: 190px; //160px;
```

Comment:
Cards seem a little bit wide for their height

Reply:
i changed it a little. 
When further change wanted: just try;)